config/example_config.toml

```diff
@@ -153,6 +153,13 @@
   # `listen` is the address to listen for incoming connections for gRPC server.
  ## listen = "127.0.0.1:50051"
 
+  # `basic_auth_credential` is basic auth for authentication layer for improve security of gRPC (https://pips.pactus.org/PIPs/pip-20),
```

Comment:
Please keep the comment shorter without a link. No need to mention PIP.
Note that we will have a "How to generate a password hash for gRPC?" article on the website soon.

Reply:
Ok